Framework/Algorithms/src/Stitch1DMany.cpp

```diff
@@ -168,18 +177,14 @@ std::map<std::string, std::string> Stitch1DMany::validateCommonInputs() {
     errors["InputWorkspaces"] = "At least 2 input workspaces required.";
 
   // Check that all the workspaces are of the same type
-  if (!m_inputWSMatrix.empty() && !m_inputWSMatrix[0].empty()) {
```

Comment:
Another way it may be empty is if InputWorkspaces is set to an array of workspace name strings, but those workspaces don't actually exist in the ADS. It will pass the validator check, but won't be added in `m_inputWSMatrix` - I overlooked this previously.